boards/matek/h743/init/rc.board_sensors

```diff
@@ -13,7 +13,11 @@ then
 fi
 
 # Internal SPI bus MPU-6000
-mpu6000 -R 12 -s start
+if ! mpu6000 -R 12 -s start
```

Comment:
Add the specific bus (-b) while you're here?

Reply:
Added it in [90adb9f](https://github.com/PX4/PX4-Autopilot/pull/20184/commits/90adb9fb6feca3ea48f3dbae94f8f1889e5a5aa4)